deluge/ui/gtk3/gtkui.py

```diff
@@ -21,13 +21,13 @@
 # isort:imports-thirdparty
 from gi.repository.GLib import set_prgname
 from gi.repository.Gtk import Builder, ResponseType
-from twisted.internet import defer, gtk3reactor
+from twisted.internet import defer, gireactor
 from twisted.internet.error import ReactorAlreadyInstalledError
 from twisted.internet.task import LoopingCall
 
 try:
     # Install twisted reactor, before any other modules import reactor.
-    reactor = gtk3reactor.install()
+    reactor = gireactor.install()
```

Comment:
This is just to avoid using a deprecated class name.

Reply:
We support a minimum version of Twisted and afaik this would break that so needs to be reverted, I don't think they are actually removing it anytime soon. We could add a Twisted version check for the import but would need a comment to explain why gireactor is being used since the Twisted docs are now contradictory about not using it for UI.

Reply:
Does it though? the current requirement spec is: 
```twisted[tls]>=17.1; sys_platform != 'win32'
twisted[tls]<23,>=17.1; sys_platform == 'win32'```

And the Twisted api documentation for 16.4.1, a full major version earlier (https://twisted.org/documents/16.4.1/api/twisted.internet.gireactor.html), already has the `gireactor`, with an API that I think is identical?